vars/deploy-on-openstack.yml

```diff
@@ -18,13 +18,9 @@ deploy_on_openstack_sesnode_userdata: |
 deploy_on_openstack_repos_to_configure:
   SLE12SP3-product: http://provo-clouddata.cloud.suse.de/repos/x86_64/SLES12-SP3-Pool/
```

Comment:
is this repo accessible by the external tech preview users? this is not related to this PR :-)

Reply:
it is obviously not accessible by externals, but thats unrelated to this PR.